docs/USAGE.md

```diff
@@ -0,0 +1,282 @@
+# Usage
+
+**WARNING**: Please notice that no official release has been created for this repo yet, so installation instructions below would require you to build the code manually beforehand.
+
+## Configure the OpenTelemetry Tracing Library for .NET
+
+### Configuration values
+
+Use these environment variables to configure the tracing library:
+
+| Environment variable | Default value | Description |
+|-|-|-|
+| `OTEL_SERVICE_NAME` |  | The name of the service. |
+| `OTEL_ENV` |  | The value for the `environment` tag added to every span. This determines the environment in which the service is available in OTEL µAPM.  |
+| `OTEL_TRACE_ENABLED` | `true` | Enable to activate the tracer. |
+| `OTEL_TRACE_DEBUG` | `false` | Enable to activate debugging mode for the tracer. |
+| `OTEL_TRACE_AGENT_URL` | `http://localhost:9080/v1/trace` | The hostname and port for a OTEL Smart Agent or OpenTelemetry Collector. |
```

Comment:
For the time being:
```suggestion
| `OTEL_TRACE_AGENT_URL` | `http://localhost:9080/v1/trace` | The URL to where send the traces. |
```
Could you please open an issue to use env vars from the exporters instead?